src/capture.js

```diff
@@ -10,7 +10,11 @@ function assertActive(id) {
   }
 }
 function assertConnected(elem) {
-  if (!elem.ownerDocument.contains(elem)) {
```

Comment:
I noticed during testing that IE10 does not support `document.contains()`. We ran into this bug once before: https://github.com/jquery/PEP/issues/320

A separate issue would probably have been more appropriate.